engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NKGlobalParameters.scala

```diff
@@ -0,0 +1,34 @@
+package pl.touk.nussknacker.engine.flink.api
+
+import java.util
+import scala.collection.JavaConverters._
+
+import org.apache.flink.api.common.ExecutionConfig
+import org.apache.flink.api.common.ExecutionConfig.GlobalJobParameters
+
+//we can use this class to pass config through RuntimeContext to places where it would be difficult to use otherwise
+//Also, those configuration properties will be exposed via Flienk REST API/webconsole
+//TODO: 
+case class NKGlobalParameters(buildInfo: String,
```

Comment:
Maybe `Nk` prefix to be concise with `Esp*` classes?

Reply:
ok

Reply:
addressed in https://github.com/TouK/nussknacker/pull/884